### PR TITLE
changes pythonpath

### PR DESCRIPTION
**Pull request by sjovdnbos**
changes in the pythonpath of the framework

### CODE REVIEW

**Comment by domsj** [comment]
nice commit message :stuck_out_tongue_winking_eye: (click the ellipsis to see it in all its glory)

File: scripts/travis/docker/ubuntu:14.04/Dockerfile [hunk 1]
```diff
@@ -29,6 +29,5 @@ ADD . /opt/OpenvStorage
 # Further tweaks to run our tests
 RUN cp /opt/OpenvStorage/scripts/system/ovs /usr/bin/ \
     && chmod 777 /usr/bin/ovs \
-    && export PYTHONPATH=$PYTHONPATH:/usr/local/lib/python2.7/dist-packages:/usr/lib/python2.7/dist-packages \
```
Comment (JeffreyDevloo) [inline scripts/travis/docker/ubuntu:14.04/Dockerfile]: This should not be removed

File: scripts/docs/generatedocs.sh [hunk 2]
```diff
@@ -20,7 +20,6 @@ if [ "$(ls -A $DIR/)" ]; then
      echo "Directory $DIR/ must be empty"
      exit 1
 fi
-export PYTHONPATH=$PYTHONPATH:/opt/OpenvStorage/lib/python2.7/site-packages/
```
Comment (JeffreyDevloo) [inline scripts/docs/generatedocs.sh]: Not sure what this does so better keep it (might have to change once we need this thing)

File: packaging/debian/debian/openvstorage-core.postinst [hunk 3]
```diff
@@ -1,3 +1,7 @@
 #!/usr/bin/env bash
 
 python /opt/OpenvStorage/scripts/install/openvstorage-core.postinst.py "__NEW_VERSION__" "$@"
+
+# create new .pth file with our path
```
Comment (JeffreyDevloo) [inline packaging/debian/debian/openvstorage-core.postinst]: Capitals :0

**Review by JeffreyDevloo — CHANGES_REQUESTED** [review]
Add webapps export to docker file

**Review by JeffreyDevloo — APPROVED** [review]

**Pull request merged**